ocs_ci/helpers/managed_services.py

```diff
@@ -313,9 +322,16 @@ def get_all_storageclassclaims():
          List: OCS objects of kind Storageclassclaim
```

Comment:
I believe the docstring should be updated with the description of the new behaviour: it returns storageclassclaims or storageclaims depending on the version, not just storageclassclaims as before the change.